std/assembly/internal/string.ts

```diff
@@ -161,51 +159,61 @@ export function compareUTF16(ptr1: usize, ptr2: usize, len: usize): i32 {
   return cmp;
 }
 
-export function stringRepeat(dest: usize, src: usize, length: i32, count: i32): void {
+export function repeatUnsafe(
+  dest: String,
+  destOffset: usize,
+  src: String,
+  length: i32,
+  count: i32
```

Comment:
What are `length` and `count`? Can't one be computed from the other?

Reply:
`length` is length of repeated source. Is better rename it to `srcLenght`?

Reply:
`count` is how many repeat times we want

Reply:
Done!